catalog-rest-service/src/main/resources/json/schema/api/data/createTable.json

```diff
@@ -31,6 +31,13 @@
       },
       "default": null
     },
+    "profileSample": {
```

Comment:
@pmbrull I was thinking adding profilerConfig at table level and store that in extension
 profiler config will contain the following
1. What columns users want to enable the profiler for, if there are none we will profile all the columns
2. regarding profileSample should be keep at table level or also at column level. i.e table level one can say at 100% and column level it can be 60%. Not sure if this is needed but for now we can go to table level

Reply:
Hi @harshach, makes sense. I'd say to start with a % at table level (note that rowCount is invariant by this, we always do the full count). That would be simpler to manage and configure at least in the beginning. Having different % on columns might make it actually less performant as we'd need multiple queries with different sample sizes while now most of the metrics are computed in a single go.

Let's keep our ears open though in case there are some use cases I might be missing for that 🤔